src/main/java/seedu/address/logic/parser/ParserUtil.java

```
@@ -108,69 +106,30 @@ public static Tag parseTag(String tag) throws ParseException {
-    }
-
-    /**
-     * Parses {@code String options} into a {@code String}.
-     * TODO: Change this to return Option Object instead
+     * Parses {@code String choice} into a {@code Choice choice}.
      */
-    public static String parseOption(String option) throws ParseException {
-        requireNonNull(option);
-        String trimmedOption = option.trim();
-        // TODO: Refactor this when the Option class is created:
-        //  Parse exception string should be in Option class, as well as the validation logic
-        //  (predicate in if statement)
-        if (trimmedOption.equals("")) {
-            throw new ParseException(MESSAGE_INVALID_OPTION);
+    public static Choice parseChoice(String choice, boolean isCorrect) throws ParseException {
+        requireNonNull(choice);
+        String trimmedChoice = choice.trim();
+        if (!Choice.isValidChoiceTitle(trimmedChoice)) {
+            throw new ParseException(Choice.MESSAGE_CONSTRAINTS);
         }
-        return trimmedOption;
+        return new Choice(trimmedChoice, isCorrect);
     }
 
     /**
-     * Parses {@code List<String> options} into a {@code String[]}.
-     * TODO: Change this to return Options Object instead
+     * Parses {@code List<String> choices, String answer} into a {@code Set<Choice> choice}.
      */
-    public static String[] parseOptions(List<String> options) throws ParseException {
-        requireNonNull(options);
-        // TODO: Refactor this when the Options class is created:
-        //  Parse exception string should be in Options class, as well as the validation logic
-        //  (predicate in if statement + cannot have duplicate options)
-        if (options.size() < 3) {
-            throw new ParseException(MESSAGE_INSUFFICIENT_OPTIONS);
+    public static Set<Choice> parseChoices(List<String> choices, String answer) throws ParseException {
+        requireAllNonNull(choices, answer);
+        if (choices.size() < NUMBER_OF_INCORRECT_CHOICES) {
```

For this, should we just throw an exception as long as the number of choices is not equals to the number of expected choices? Rather than just not saving additional choices if the user inputs more than three.